science/hacktv/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        fsphil hacktv b3d9ecb87f042bb6729824fabb40d55d145c3e65
+version             0.1.20221001
+
+categories          science
+license             GPL-3+
+maintainers         {@kesterlester cern.ch:lester} \
+                    openmaintainer
+description         Broadcast TV using an SDR like HackRF One
+long_description    Broadcast TV using a Software Defined Radio like HackRF One
+
+checksums           sha256  9d2f65aa25427983192331c3cf0b1e88702c0392e02dec5184bb8efcb5657563 \
+                    rmd160  6a64cf2f5fe7f4b2ab1fe492c9441b31327136bd \
+                    size    255617
+
+patchfiles          destdir-fix.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:ffmpeg-devel \
+                    port:SoapySDR \
+                    port:hackrf
+
+PortGroup           makefile 1.0
+# Above line alegedly does things including:
+# build.args-append       CC=${configure.cc} \
+#                         CXX=${configure.cxx} \
+#                         CPP=${configure.cpp}
```

Comment:
Please, remove useless comments

Reply:
Really?  I put them there as I thought I might have to maintain this in future, and 6 months from now I'll never remember what defines CC unless I have a comment like that in there.  I mean, if someone else maintains it and does loads of Portfile maintaining they wouldn't need that comment. But realistically I will need it -- or rather it will drastically reduce the time I spend having to work out what to do.

Reply:
@reneeotten?

Reply:
I don't see a point for these comments either. The `makefile` PortGroup defines these things so you will not need to remember, that's the whole point.